client/src/containers/overview/table/utils.ts

```diff
@@ -13,14 +18,14 @@ const OMITTED_FIELDS = [
 ];
 
 export const filtersToQueryParams = (
-  filters: z.infer<typeof filtersSchema>,
+  filters: z.infer<typeof filtersSchema | typeof scorecardFiltersSchema>,
```

Comment:
These query params are the ones sent to filter the tables and, as far as I know, we are applying the same filters, the global ones (country, project size, carbon type... ), to the three tables, so it shouldn't be needed extending the type of `filters` here.